src/api/app/models/bs_request.rb

```diff
@@ -107,6 +108,20 @@ def check_creator
     errors.add(:creator, "Login #{user.login} is not an active user")
   end
 
+  def double_actions
```

Comment:
Yeah while double is true duplicate is the more precise term to use here IMO. I would also call the method some more descriptive i.e. `def has_no_dupcliate_actions`.